prow/plugins/help/help.go

```diff
@@ -30,12 +30,15 @@ import (
 const pluginName = "help"
 
 var (
-	helpLabel         = "help wanted"
-	helpRe            = regexp.MustCompile(`(?mi)^/help\s*$`)
-	helpRemoveRe      = regexp.MustCompile(`(?mi)^/remove-help\s*$`)
-	helpGuidelinesURL = "https://git.k8s.io/community/contributors/devel/help-wanted.md"
-	helpMsgPruneMatch = "This request has been marked as needing help from a contributor."
-	helpMsg           = `
+	helpLabel                  = "help wanted"
+	goodFirstIssueLabel        = "good first issue"
+	helpRe                     = regexp.MustCompile(`(?mi)^/help\s*$`)
+	helpRemoveRe               = regexp.MustCompile(`(?mi)^/remove-help\s*$`)
+	helpGoodFirstIssueRe       = regexp.MustCompile(`(?mi)^/help good-first-issue\s*$`)
+	helpGoodFirstIssueRemoveRe = regexp.MustCompile(`(?mi)^/help remove-good-first-issue\s*$`)
+	helpGuidelinesURL          = "https://git.k8s.io/community/contributors/devel/help-wanted.md"
+	helpMsgPruneMatch          = "This request has been marked as needing help from a contributor."
```

Comment:
It may be good to change this comment to better reflect what these labels mean. We aren't flagging it as "needing help", but instead flagging it as looking for contributions or something like that...

Maybe `This issue has been marked as suitable for new contributors.`?

Reply:
I had added a `goodFirstLabelMsg` before. Maybe I should add it back?

We can have a separate comment added when the `good first issue` label is added (and pruned when removed).

The guidelines link can point to the sub-heading of "Good First Issue" in the help-wanted doc or remain the same as that of help wanted. Wdyt?

Reply:
I don't think that we need to use separate messages for each. I believe that we agreed to use the default GitHub labels in the same spirit as they are used across GitHub? So I am more concerned that the current message "marked as needing help from a contributor" reinforces a behavior that we actively do not want, namely labeling issues that are hard and we need more help solving with "help wanted".

On sig service catalog, we have another label "needs-help" to indicate that we are stuck. 😀 I Noticed that k8's has a `¯\_(ツ)_/¯` label, maybe that means the same thing?

Reply:
I agree with @carolynvs 's "suitable for new contributors" message.

Reply:
Done. Ptal.